src/markdown.c

```diff
@@ -1413,6 +1413,25 @@ prefix_quote(uint8_t *data, size_t size)
 	return 0;
 }
 
+static size_t
+prefix_blockspoiler(uint8_t *data, size_t size)
```

Comment:
Your assessment matches my understanding of that code - should I add some comments describing that in there?